### PR TITLE
[HOPSWORKS-888] Bugfix: Infer tf-record-schema does not work with bigint

### PR DESCRIPTION
**Pull request by Limmen**

### CODE REVIEW

**Comment by Limmen** [comment]
@tkakantousis

File: src/main/java/io/hops/util/Hops.java [hunk 1]
```diff
@@ -1898,6 +1905,9 @@ public static void createFeaturegroup(
     if (numClusters == null) {
       numClusters = 5;
     }
+    if(jobName == null || jobName.isEmpty()){
```
Comment (tkakantousis) [inline src/main/java/io/hops/util/Hops.java]: @Limmen we usually call the google `Strings` library for such checks, [example](https://github.com/logicalclocks/hops-util/blob/release-0.8/src/main/java/io/hops/util/KafkaProperties.java#L85).

File: src/main/java/io/hops/util/exceptions/InferTFRecordSchemaError.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.hops.util.exceptions;
+
+/**
+ * Exception thrown when a the TF-record schema of a spark dataframe could not be inferred
+ * be created
```
[the remark this answers is not in the document]
Comment (Limmen) [reply src/main/java/io/hops/util/exceptions/InferTFRecordSchemaError.java]: Done

File: src/main/java/io/hops/util/Hops.java (hunk 1 above)
Comment (Limmen) [reply src/main/java/io/hops/util/Hops.java]: Done

**Pull request merged**